pinot-perf/src/main/java/com/linkedin/pinot/perf/QueryRunner.java

```diff
@@ -213,7 +213,7 @@ public void run() {
             }
             long startTime = System.currentTimeMillis();
             try {
-              System.out.println(driver.postQuery(query));
+              driver.postQuery(query);
```

Comment:
May want to add a verbose switch later so we can also see the queries if required.